recipes/commonregex/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "commonregex " %}
+{% set version = "1.5.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/madisonmay/CommonRegex
+  git_tag: master
+  git_depth: 1
```

Comment:
This, I'm afraid, is not going to be allowed.  It would also be an absolute nightmare to maintain (which is why it's not allowed).

Reply:
Could you ask the repo owner(s) to make tags each time they update the version number?

Reply:
Thank you for the help @xylar - let me try.
https://pypi.org/project/commonregex/#history
The last update to this project on pypi was in 2014 ! https://pypi.org/project/commonregex/1.5.4/
So I don't know how much luck I will have with the maintainers, I might have better luck with the OpenMetadata team, if they are interested in supporting conda - this one package might be an issue they might want to solve for on their end. 

One thing I did notice is they do have a packaged distributable on repo
https://github.com/madisonmay/CommonRegex/tree/master/dist
But I could not figure out how to get an static SHA from it, and I was wondering if that was allowed anyway - its not a tag or a release... just packaged and put into a folder.

Reply:
Can you get the source from PyPI instead?

Reply:
> One thing I did notice is they do have a packaged distributable on repo

No, I don't think that solves the problem.

Reply:
I take that back.  I think you could use:
https://raw.githubusercontent.com/madisonmay/CommonRegex/master/dist/commonregex-1.5.5.tar.gz
as the source URL.  It's not great but it's what there seems to be.  As long as you get the sha256 for that file:
```
08e93aec3ce2a3ae29d89cc249af47e70df10218f2161455b3df19fb5cb67d00
```
and it stays the same, I think that should work.

Reply:
```suggestion
  url: https://raw.githubusercontent.com/madisonmay/CommonRegex/master/dist/commonregex-1.5.5.tar.gz
  sha256: 08e93aec3ce2a3ae29d89cc249af47e70df10218f2161455b3df19fb5cb67d00
```